gcloud/bigtable/column_family.py

```diff
@@ -238,6 +242,27 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
+    def update(self):
+        """Update this column family.
+
+        .. note::
+
+            The Bigtable Table Admin API currently returns
+
+             ``BigtableTableService.UpdateColumnFamily is not yet implemented``
```

Comment:
@tseaver I just tried to catch the error in the system tests and found that this method **has been implemented**, so that's good. Will update this PR to reflect that. (Also added a system test to verify that `Table.rename` remains unimplemented.)